packages/discord.js/src/index.js

```diff
@@ -15,6 +15,7 @@ exports.ActivityFlagsBitField = require('./util/ActivityFlagsBitField');
 exports.ApplicationFlagsBitField = require('./util/ApplicationFlagsBitField');
 exports.BaseManager = require('./managers/BaseManager');
 exports.BitField = require('./util/BitField');
+__exportStar(require('./util/Channels.js'), exports);
```

Comment:
We should not export createChannel to end users I'd say... Or should we?

Reply:
I say we shouldn't. It's for internal use only and the constructors are not documented for the individual channels.